pkgs/development/libraries/zchunk/default.nix

```diff
@@ -1,6 +1,7 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, fetchpatch
```

Comment:
No longer required now that the patch is checked in

Reply:
Removed, thanks